### ISSUE TITLE
Nav unification: Stats sparkline with little activity nearly invisible/doesn't look right on certain themes 

### ISSUE THREAD

**Issue by sarayourfriend** [Automattic/wp-calypso#49133]
### Steps to reproduce the behavior

1. Go to `https://wordpress.com/stats/day/:site`
2. Observe the sparkline that appears on the "stats" line for a site that has some but not much activity (this doesn't happen for every site, not sure what determines whether it appears, feel free to SU to me and observe on `https://wordpress.com/stats/day/sarayourfriend.wordpress.com`)
3. Switch to the "aquatic" or "ocean" theme

#### What I expected to happen
The 'sparkline' should be clearly visible on all themes

#### What actually happened
The 'sparkline' is almost invisible on many themes or otherwise looks incorrect/artifacted

#### Browser / OS version

Firefox, macOS Big Sur

#### Is this specific to the applied theme? Which one?
Tested "aquatic", "ocean", "contrast" and a couple other themes

#### Does this happen on simple or atomic sites or both?
Not sure what triggers the sparkline but I only see it for a simple site.

#### Is there any console output or error text?
N/A

#### Level of impact (Does it block purchases? Does it affect more than just one site?)
Very low/minimal

#### Screenshot / Video: If applicable, add screenshots to help explain your problem.
<img width="292" alt="Captura de Tela 2021-01-21 às 09 33 55" src="https://user-images.githubusercontent.com/24264157/105388827-f7dee200-5bcb-11eb-9637-132d7069f65e.png">
<img width="283" alt="Captura de Tela 2021-01-21 às 09 30 20" src="https://user-images.githubusercontent.com/24264157/105388829-f8777880-5bcb-11eb-909d-d681d0c077fd.png">
<img width="294" alt="Captura de Tela 2021-01-21 às 09 33 43" src="https://user-images.githubusercontent.com/24264157/105388831-f9100f00-5bcb-11eb-9a2e-599c97ba7151.png">

**Comment by pablinos** [comment]
Hi @sarayourfriend! Both @retrofox and I have had a look at this, and we're struggling to replicate it. The sparkline colour is the same as the icon colour on the left

<img width="275" alt="image" src="https://user-images.githubusercontent.com/96462/112354018-33795380-8cc4-11eb-8eb8-fc744b1aeb4e.png">

I can see things looking a little washed out in general, and that the sparkline is not the same colour in your tests. The sparkline is always showing for me, so it might be related to what's causing it not to display for you. Could there be anything else that is affecting the colour for you? Does it look the same in another browser for example? I use Firefox too, but it could be that something is cached.

**Comment by sarayourfriend** [comment]
I think it may have been fixed by https://github.com/Automattic/wp-calypso/pull/49988

**Comment by pablinos** [comment]
I think you're right. Fantastic. I'll close this then.
